godotproject/Menus/InGameMenu/InGameMenu.gd

```diff
@@ -60,7 +60,7 @@ func _screen_should_fade(screen):
 	return screen != null && screen.fade_transition
 
 func navigate_to_new_screen(new_screen):
-	if _screen_should_fade(screen_current) || _screen_should_fade(new_screen):
+	if new_screen != null && (_screen_should_fade(screen_current) || _screen_should_fade(new_screen)):
```

Comment:
This causes us not to play the fade effect if we're exiting the menu. This should have never happened previously, but we now navigate to screen `null` when we're doing the fade animation to reset the menu, so I added an extra check here just in case.